canonical-websites/snapcraft.io
# Add embeddable cards to Publicise snap publisher page

Fixes #1601

Adds basic embeddable cards to Publicise page.

### QA

- ./run or https://snapcraft-io-canonical-websites-pr-1621.run.demo.haus/
- go to Publicise page of any snap
- you should see side navigation on the left side
- go to "Embeddable cards"
- page should open with embedded card preview and code to copy
- copy the code to test it out (make sure to change "snapcraft.io" in the iframe URL to demo URL: https://snapcraft-io-canonical-websites-pr-1621.run.demo.haus/)
- check if iframe renders as expected

<img width="1036" alt="screenshot 2019-02-20 at 13 30 30" src="https://user-images.githubusercontent.com/83575/53092833-12a70880-3516-11e9-84af-8f3b7c503552.png">
